environment.yml

```diff
@@ -8,10 +8,11 @@ dependencies:
     - pooch>=0.5
     - xarray
     - pandas
+    # Development requirements
+    - verde
```

Comment:
Do we really need to pull in `verde`? It seems like it's just for the `maxabs` function, which we could do without.

Reply:
You're right, we can ditch Verde as a dependency and compute the maxabs trough Numpy.